packages/tasit-sdk/src/ethers.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect, assert } from "chai";
+import { ethers } from "ethers";
+
+const ropstenProvider = ethers.getDefaultProvider("ropsten");
+const privateKey =
+  "0x18bfdd05c1e63e4a27081352e2910c164a4d34588f8d7eecfdfcb654742934c2";
+const ensSample = {
+  provider: ethers.getDefaultProvider(),
+  name: "registrar.firefly.eth",
+  address: "0x6fC21092DA55B392b045eD78F4732bff3C580e2c"
+};
+const contractABI = [
+  "event ValueChanged(address indexed author, string oldValue, string newValue)",
+  "constructor(string value)",
+  "function getValue() view returns (string value)",
+  "function setValue(string value)"
+];
+const contractAddress = "0x2bD9aAa2953F988153c8629926D22A6a5F69b14E";
+
+var wallet, randomWallet, contract;
+var rawTx = {
+  nonce: 0,
+  gasLimit: 21000,
+  gasPrice: ethers.utils.bigNumberify("20000000000"),
+  to: "0x88a5C2d9919e46F883EB62F7b8Dd9d0CC45bc290",
+  value: ethers.utils.parseEther("0.0000001"),
+  data: "0x",
+  chainId: ethers.utils.getNetwork("ropsten").chainId
+};
+
+var waitForEvent = async (eventName, expected) => {
+  return new Promise(function(resolve, reject) {
+    contract.on(eventName, function() {
+      var args = Array.prototype.slice.call(arguments);
+      var event = args.pop();
+      event.removeListener();
+      expect(args).to.deep.equal(expected);
+      resolve();
+    });
+  });
+};
+
+describe("ethers.js", function() {
+  before(
+    "should create a funded wallet from priv key and provider",
+    async () => {
+      wallet = new ethers.Wallet(privateKey, ropstenProvider);
+
+      expect(wallet.address).to.have.lengthOf(42);
+      expect(wallet.provider).to.be.not.undefined;
+
+      let balance = ethers.utils.bigNumberify(await wallet.getBalance());
+      let zero = ethers.utils.bigNumberify(0);
+
+      assert(balance.gt(zero), "wallet balance is zero");
+    }
+  );
+
+  // it("should create a wallet from priv key and provider", async function() {
+  //   wallet = new ethers.Wallet(privateKey, ropstenProvider);
+  //   expect(wallet.address).to.have.lengthOf(42);
+  //   expect(wallet.provider).to.be.not.undefined;
+  // });
+  //
+  // it("wallet should has funds", async function() {
+  //   let balance = ethers.utils.bigNumberify(await wallet.getBalance());
+  //   let zero = ethers.utils.bigNumberify(0);
+  //
+  //   assert(balance.gt(zero));
+  // });
+
+  it("should create a random wallet", async function() {
+    randomWallet = ethers.Wallet.createRandom();
+    expect(randomWallet.address).to.have.lengthOf(42);
+  });
+
```

Comment:
```suggestion
// Note: We're intentionally not testing the `fromEncryptedJson` or `encrypt` functions
// from `ethers.js` because we don't plan to expose that functionality in the Tasit SDK.
// For a detailed explanation of why, see this GitHub issue:
// https://github.com/tasitlabs/TasitSDK/issues/24#issuecomment-443576993```